ci/docs/build.sh

```diff
@@ -28,6 +28,10 @@ gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
 
+# Using absolute path of python executable to avoid obtaining wrong version of package
+gpuci_logger "Install pip only package"
+/opt/conda/envs/rapids/bin/python -m pip install sphinxcontrib-moderncmakedomain
```

Comment:
```suggestion
pip install sphinxcontrib-moderncmakedomain
```

any reason this can't be just `pip install...`? seems like that should be fine since the conda environment is activated a few lines above.

Reply:
`which python` in the build script was pointing to a host level python executable and not the python executable inside the `rapids` environment. It caused the install to acquire a `py3.8` version of `sphinxcontrib-moderncmakedomain` inside a `python3.7` container.